Sources/Sentry/SentryFileManager.m

```diff
@@ -787,7 +811,39 @@ - (void)createPathsWithOptions:(SentryOptions *)options
 
     NSError *error;
     SENTRY_CASSERT([fm removeItemAtPath:path error:&error],
-        @"Failed to remove launch profile marker file: %@", error);
+        @"Failed to remove launch profile config file: %@", error);
+}
+
+void
+removeAppLaunchProfilingConfigBackupFile(void)
```

Comment:
`l`: This is almost the same as `removeAppLaunchProfilingConfigFile`. Maybe we can reuse some code here. The FileManager already has `removeFileAtPath`. Maybe we could reuse that.

Reply:
I'll put this in this PR unless you'd like otherwise.

Reply:
Yes, please do this in this PR. An extra PR would be unnecessary overhead 👍